components/brave_new_tab_ui/components/default/editCards/style.ts

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'brave-ui/theme'
+import palette from 'brave-ui/theme/colors'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  cursor: pointer;
+  margin-top: 5px;
+  border-radius: 8px;
+  backdrop-filter: blur(23px);
+  padding: 12px 20px 8px 20px;
+  background: linear-gradient(90deg, rgba(33, 37, 41, 0.6) 0%, rgba(33, 37, 41, 0.24) 100%);
+
+  &:hover {
+    background: rgba(33, 37, 41, 0.5);
+  }
+
+  &:active {
+    outline-offset: -1px;
+    outline-style: solid;
+    outline-color: ${p => p.theme.color.brandBrave};
+    outline-width: 1px;
+  }
+`
+
+export const StyledTitle = styled<{}, 'div'>('div')`
+  width: fit-content;
+  font-size: 13px;
+  font-weight: 600;
+  color: ${palette.white};
+  margin: 0 auto;
+  font-family: ${p => p.theme.fontFamily.heading};
+`
+
+export const StyledEditIcon = styled<{}, 'div'>('div')`
+  width: 24px;
+  height: 24px;
+  display: inline-block;
+  vertical-align: text-top;
+`
```

Comment:
Seeing a couple issues with the styling:

1. The image and the text aren't center aligned together. Flexbox align-items: center would achieve that, but that leads me to...

2. The icon is defined here as 24x24 but the SVG is hardcoded to 14x16 which is displaying at the top-left of this 24x24 box. If you do want it at 24x24 then you could make the SVG scalable and display at 100% x 100% of this StyledEditIcon container or, if you want it to be 14x16, then let it be the normal SVG size

![image](https://user-images.githubusercontent.com/741836/98900676-1a83aa80-2467-11eb-8318-7b19c911b7e2.png)

Reply:
Fixed
<img width="319" alt="Screen Shot 2020-11-12 at 2 06 49 PM" src="https://user-images.githubusercontent.com/8732757/98997249-6ed36c80-24f1-11eb-9092-ccde3484e389.png">